media/js/firefox/new/pixel.js

```diff
@@ -8,12 +8,16 @@
     // pixel status bug https://bugzilla.mozilla.org/show_bug.cgi?id=1311423
     function addPixel() {
         if (!window._dntEnabled()) {
-            var $pixel = $('<img />', {
-                width: '1',
-                height: '1',
-                src: 'https://servedby.flashtalking.com/spot/8/6247;40428;4669/?spotName=Mozilla_Download_Conversion'
-            });
-            $('body').append($pixel);
+            var href = $('#strings').data('trackingPixel');
```

Comment:
TIL: dashes are converted to camelCase in dataset. https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset#Name_conversion